lib/node/http.js

```diff
@@ -255,6 +255,27 @@ class HTTP extends Server {
       res.json(200, block.getJSON(this.network, view, height, depth));
     });
 
+    // Block Header by hash/height
+    this.get('/header/:block', async (req, res) => {
+      const valid = Validator.fromRequest(req);
+      const hash = valid.uintbrhash('block');
+
+      enforce(hash != null, 'Hash or height required.');
+      enforce(!this.chain.options.spv, 'Cannot get block header in SPV mode.');
+
+      const block = await this.chain.getBlock(hash);
```

Comment:
Might be more efficient to `getEntry()` and then use `Headers.fromEntry()`. 

This will work on SPV mode as well.

Reply:
Indeed, there is no need to read and parse the entire block from disk.

Reply:
Good call, thanks for the feedback @pinheadmz @braydonf